python/MRzeroCore/phantom/voxel_grid_phantom.py

```diff
@@ -193,14 +197,23 @@ def load(cls, file_name: str) -> VoxelGridPhantom:
                 size = torch.tensor(data['FOV'], dtype=torch.float)
             except KeyError:
                 size = torch.tensor([0.192, 0.192, 0.192])
-        
+
+            try:
+                Masks = {}
+                tissues = [key[5:] for key in data.keys() if key.startswith('mask_')]
+                for tissue in tissues:
+                    mask = torch.tensor(data[f'mask_{tissue}'], dtype=torch.uint8)
+                    Masks[tissue] = mask
+            except KeyError:
+                print("No masks found")

```

Comment:
There can't be a key error when iterating over keys, this code could be shortened to:

```python
masks = {key: torch.tensor(mask) for key, mask in data.items() if key.startswith('mask_')}
```

Also, depending on your choice of data type in the brainweb generation, this might be a float32 (or do not set the data type explicietley)

Reply:
thank you, the latest changes use keys that start with `tissue_`  (tissue_gm, tissue_wm, etc) without explicit datacasting. the output masks are tensors of float32